### ISSUE TITLE
Bug report: Test on Theos

### ISSUE THREAD

**Issue by superstar54** [aiidalab/aiidalab-qe#340]
Computer : cscs-daint
- set up computer
- set up code pw, successfully.
- set up code dos, crash. (pre-set some parameters from database by quick-setup first.)

## Additional comments (optional):

_Example: I submitted a band structure calculation for Silica._

## Attachments

<details>
<summary>Traceback</summary>

```python-traceback
/opt/conda/lib/python3.9/site-packages/traitlets/traitlets.py in set(self, obj, value)
    687 
    688     def set(self, obj, value):
--> 689         new_value = self._validate(obj, value)
    690         try:
    691             old_value = obj._trait_values[self.name]

/opt/conda/lib/python3.9/site-packages/traitlets/traitlets.py in _validate(self, obj, value)
    721             value = self.validate(obj, value)  # type:ignore[attr-defined]
    722         if obj._cross_validation_lock is False:
--> 723             value = self._cross_validate(obj, value)
    724         return value
    725 

/opt/conda/lib/python3.9/site-packages/traitlets/traitlets.py in _cross_validate(self, obj, value)
    727         if self.name in obj._trait_validators:
    728             proposal = Bunch({"trait": self, "value": value, "owner": obj})
--> 729             value = obj._trait_validators[self.name](obj, proposal)
    730         elif hasattr(obj, "_%s_validate" % self.name):
    731             meth_name = "_%s_validate" % self.name

/opt/conda/lib/python3.9/site-packages/traitlets/traitlets.py in __call__(self, *args, **kwargs)
   1130         """Pass `*args` and `**kwargs` to the handler's function if it exists."""
   1131         if hasattr(self, "func"):
-> 1132             return self.func(*args, **kwargs)
   1133         else:
   1134             return self._init_call(*args, **kwargs)

/opt/conda/lib/python3.9/site-packages/ipywidgets/widgets/widget_selection.py in _validate_value(self, proposal)
    241             return findvalue(self._options_values, value, self.equals) if value is not None else None
    242         except ValueError:
--> 243             raise TraitError('Invalid selection: value not found')
    244 
    245     @observe('value')

TraitError: Invalid selection: value not found
```
</details>

<details>
<summary>Environment fingerprint</summary>
<pre>eNptWF1v47gV_SuB-7oyLMl24nmbDtCHxU6bNrsdFEURUBItcyyRDEnFURbz3_d-0VaKffQhRd6Pc8-99O-rVx2icXb1qfxp5QeV
ji6Mq0-_r1RoTybpNk1Brz79d7XfNiatflqt_gf75nRy9vn66apeH9ZlDas36C_VZlv81kw2TXdPXx_v_hbM3eepv6v2d-XhU7X5
VJV3v_365a7aVNXqB5yp2rPqdaS7jelU0Tq8eVWtq3UJRzPoo546B_Bm_bDeXOGXScFFo44-6Bhxfbsub-uDavAkOOgjWOjB0lnl
ulrCJzfqP__gJeMLoxgvLi6c25My9s93XEzX6xSLRkV2a7PeNHy2K7w5KwD34BR_BFl4ocDWDAx6bEwLSJm32GiqUxoHwegkOxtH
X-15j_edCZF2bNdbRkbX6WX8MM-vuvAqnQjecihCzz-rff7tbFW0x6NB40uwa_MRLhpjO2P7yOsVr5Ov9bqSYMSU3FnbKAF4IGy2
bdFrq4NKLpCxJX2bEtmO39Jhf1WNHujLcrNGtjXAmVYNAxtKexDyLqS4Pk62Tc4NsRjCVLSqPWk6e0-BaNow-8S2kYONVlMyx2mI
bvJbJlBJRjeDho8B2YHFDBh71mzpDn-DSyeya8tmBZcG42cy657M-oLXf3E2BcfWIjvwLLKLzKQrKzKuXbjObrU6gHEU-g1z6z6j
ck9JV0t-wPAdmQp8DFGnwgKf1GDeyWqMJy_q0dijkxMOfJMJ7RT5SGZIO5j2DMBDvgN_Q3Xao-mLoxmYTfvb3iJ6Yy1dhXZRMls3
uKBGJRTbEWQ7xU7K1QgUIgTFSdkOAt2TO9cNcKdrvksyHhhLbgrkA9CE44K5dX1Q_oRwfcBaQ3xuh2wWU6pTLYic8IKBpNpBxag5
BnhFp5uplwv2FIJOgzgJW3cSzU4fp6i7t3GQvDMIggTpZE3MRO5MPBfgBqhrTKxyOYCdjayutJ2LrJuGi2mlHFmpNOiBtkmKtlrv
yCw9KjMUr5Do7lpKrB96HIEFbb6rfiCewQkQKWdsYmcJe2u1R3v7AIXAXCIr9JtuoUIkIezzEQpaecOROlyh79HZiOxS7DRLCFJl
cESlWhJ6HEx6FqtqCdkREnoriJo5eJy4DzEr0VfK7qt556hwYfZBazvoJBbittM0KnsMRgOZKIPMBNNZRXeiy2ZExRhMU0CQFDIg
l__HVWgtwLSWmLGDwsTzofxACizp0h493TA4-kGKqmJAcnq_ruu8ByEUPojqEBcahkvgiSDyvbQPOuL-_8BCvyVwUHeL1JgUO2V7
DVmMQj386LvuTA4jbvvZ2O-qolDIBgfy1tBBbA0mcye-7OQ3cUa0pBYs6CNbLL-vBKjzQRP4pcMzaAQQj2PBJ-YFJsJVeAUvog6v
Irfl4cNKojoAEovULK45TQ3dvVmCPAdA1qmub2Dh5x7rKS5iflt9vhpQoa5-XCxuicmhP5vLTf3vBYhuyE5wKx5UOIPYgTyH3BE2
BCcoQejRrfRIPBEyMqqxUSJC7CjW0lV-SzZ6cP0UpoWefFVnJ8nERNEpizO-ghGdu1iJSinQ5J_UUS8MGFXyA3a1RsaL-gNYGAtS
LZas97hmYppsPgL9HaGu59zADusDQUEXoL8h1ztwckt2jb7IulJviCSjH1nsSpkHx9hjqxDZpxumIZnOtIlqkTGrUoTypbM5wLYh
fadhKrcixISUuWMD5CxkjInKV9oGp2OVqP55xLI6poJGGMNx3pH7ViccB9-Yggj0w6vRF6EkRs-6pBvnzmQshL5aYEU8mXHBaTuN
kvqKk-RgWDnlbLBxDlR7VJ1etu6MFVf3JPolhp87LSt6xXLpYdBMSTWDluFzQ-CoByWBFuDFLxqUh1atorhfC-
BaICfkNuO8EWYAQ4ysMuGxBPJAz9-mU-
ZDHsy9fvOa8rqVsdX3cZJipY9g7Bh0hGlHSw4xrY9mGBzG_LBmWj0ayXHJ2fPG8wDCZPHnHrX4GYUeqvXZqlHnJkqXIttn7vni-S
CZoc5MlgR4O6STnmKx4FTJacNFnwp072xSJgOtRLyZjuaW4ePcOt9XFKoDHy0QTtqKNO-6kmY4GjpTXDDYQ8csNEwDC1HzcwPywo
CEZP5ijv_6RqHlYvBwS1RJRKRm5CpUcE7J0BQGqo0tj5p-hn6T5I1Sch94vMlqlUP2OP_87VcZWsjMGSoKuuCNnHTB4_x1fvrnL9
J4akL-rr4MSzbN__DaPj3hpiorvZ_RVCY1YkT0GZCYrsng1vc4P4GCMj3vxQVqyTABaMlGBXSrbivY1wrQ2F5Cscmfue-tGv2iYr
GFKyYGDxkAvLOLpXjzn89ff2GlouV3evBhVuhQeFDmKqjEtaBfJpAbhip2ImPXBw5G62G5MDoRSRl9wrGtDw_3wmvaOmGBr2clo2
t5zylY4OuWxQZ5tL8uFbhU5Dl-g83xAa-IrQF6F_BuCjZzrWI8dzC2JWp99lnWD9Qrn2CQqX4NKp4WD1t4wkw-h2O_k04VT6Ds02
Q6kSZ6UEXzxjdwXKM18CRy4i0hMNZGo1-1CFi19i4m-tT3efSBr7EUgICfB5xj5ty4aR88Ec6FzInQndYCBtROGdtY22IK0I4KDH
ldH7g2EZ6znMukA4o7YeMXGSSxAbri42SWlxdDAR5sSv7_EA_TKWjVeQhOmwYZ5Ag3dm5jrBaDSXI5w1ye-NssiIwRfl8OI_CIkO
t4hE8v3UhKsecMAMdNGnj62YnGghRB9T3TSIr_BUWRFlybsIBoL1cwPNGhwypQZUNTNdqOAZr8EdhfyO48RYPg3B6ykpCacchbzf
1xzzT4t4KpfV60kItK7alzffYfa_LSwvAmf3tUlPCLbuilKuNcxZC2MJDJXxs53wBHqCx8VWWZL4WX33Q4v-upX4zdl5PWHPqapY
anxqKBRhmxrcjJm-vSs9fBOJxmrt24XCxH21wjvPjH5wKvInn-sC3wHL0O4kQ3DNisQv7bCI17N95fH2U_fvwBqRmoDA==</pre>
</details>

**By submitting this issue I confirm that I am aware that this information can
potentially be used to determine what kind of calculation was performed at the
time of error.**

**Comment by unkcpz** [comment]
Guess maybe addressed by https://github.com/aiidalab/aiidalab-widgets-base/issues/416.

**Comment by unkcpz** [comment]
The way to reproduce if I remember correctly:
1. setup the computer and code of pw.x first
2. Click the `Setup Code` button of dos.x code setup widget.
3. 🔥 break

**Comment by unkcpz** [comment]
I release the beta version `23.1.0b0`, I test locally it is all fine to set new codes on cscs. You can try to delete all cscs codes, and update qeApp and then test it again.

**Comment by unkcpz** [comment]
This is fixed by #342
